test/image/mocks/gl2d_parcoords.json

```diff
@@ -62,6 +62,8 @@
       },
       {
         "label": "Min height width",
+        "range": [200000, -30000],
+        "constraintrange": [0, 100000],
```

Comment:
This is covered by the `gl2d_parcoords_constraints` mock too, but I added a reversed-axis constraint to this during my tests.